### ISSUE TITLE
🐛 Bug: Use of lengthy text in form input label & help text

### ISSUE THREAD

**Issue by dstala** [nocodb/nocodb#5438]
### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

Use of lengthy text in 
  - Form input label - text gets truncated
  - Form help text - overflows

### Desired Behavior

text should wrap around & gets displayed in multiple lines

### Project Details

Node: **v16.14.0**
Arch: **arm64**
Platform: **darwin**
Docker: **false**
RootDB: **sqlite3**
PackageVersion: **0.106.0-beta.1**

### Attachments

![Screenshot 2023-04-10 at 7 54 02 AM](https://user-images.githubusercontent.com/86527202/230813167-a3283243-111c-49be-b3bb-68611ea66212.png)

**Comment by dstala** [comment]
@Beejay-bot assigned to you

**Comment by ilyak0** [comment]
The initial request was by me, as per @dstala's request I'm copying here the setup details in case it's needed.

Node: **v16.19.1**
Arch: **x64**
Platform: **linux**
Docker: **true**
RootDB: **mysql2**
PackageVersion: **0.106.0**

**Comment by dstala** [comment]
@ilyak0 @Beejay-bot if you have something on this, please raise a PR;

**Comment by nith2001** [comment]
Figured I could pick this up, hope that's okay. New to the codebase so I'll be pretty slow on solving this, so feel free to air me and solve this quicker in case it's a higher priority.

I noticed that this Vue tag [here](https://github.com/nocodb/nocodb/blob/develop/packages/nc-gui/components/smartsheet/Form.vue#L680-L695) displays the question title of the form, and [this](https://github.com/nocodb/nocodb/blob/develop/packages/nc-gui/components/smartsheet/Form.vue#L746) is where the question description is. 

I'm guessing some mild CSS styling needs to be done to fix the issues mentioned above to the title and description of each form question, but I'm new to Vue and CSS in Vue, so that'll take me a minute. Will keep working on it
